### ISSUE TITLE
Divide by zero in tICA

### ISSUE THREAD

**Issue by kyleabeauchamp** [msmbuilder/msmbuilder-legacy#369]
I was comparing a bunch of tICA models and I saw this:

```
/home/kyleb/opt/lib/python2.7/site-packages/scipy/linalg/decomp.py:59: RuntimeWarning: divide by zero encountered in true_divide
  w = (alphar + _I * alphai) / beta
/home/kyleb/opt/lib/python2.7/site-packages/scipy/linalg/decomp.py:59: RuntimeWarning: invalid value encountered in true_divide
  w = (alphar + _I * alphai) / beta
```

I also think one of my eigenvalues is blowing up.  This is very low priority, as it happened only once out of very many models.  I figured I'd report it though.
